package.json

```diff
@@ -18,7 +18,7 @@
     "format": "run-s format:package format:prettier format:eslint",
     "format:eslint": "eslint --cache --fix ./ >/dev/null 2>&1 || true",
     "format:package": "prettier-package-json --write",
-    "format:prettier": "prettier --write '**/*.{js,json,md,yml}' '.editorconfig' 'LICENSE'",
+    "format:prettier": "prettier --write \"**/*.{js,json,md,yml}\" \".editorconfig\" \"LICENSE\"",
```

Comment:
Hey @kripod - just curious on the reason for the change here? Was the single quotes causing problems?

Reply:
Yeah, it didn’t run on Windows without the change. The `format:eslint` script still doesn’t run due to the usage of `/dev/null`.

Reply:
Cool, good to know 👍